server/graphql/cache.js

```diff
@@ -1,176 +1,27 @@
-import { difference, isEqual } from 'lodash';
-
-import { TransactionKind } from '../constants/transaction-kind';
 import { md5 } from '../lib/utils';
 
-export const purgeCacheForCollectiveOperationNames = [
-  'CollectivePage',
-  'ContributePage',
-  'BudgetSection',
-  'BudgetSectionWithHost',
-  'UpdatesSection',
-  'TransactionsSection',
-  'TransactionsPage',
-  'ExpensesPage',
-  'CollectiveBannerIframe',
-  'CollectiveCover',
-  'RecurringContributions',
-  'ContributionsSectionStatic',
-  'Members_Users',
-  'Members_Organizations',
-];
-
-export function checkSupportedVariables(req, variableNames) {
-  if (!req.body.variables) {
-    return false;
-  }
-  // Check missing variables
-  if (difference(variableNames, Object.keys(req.body.variables)).length > 0) {
-    return false;
-  }
-  // Check extra variables
-  if (difference(Object.keys(req.body.variables), variableNames).length > 0) {
-    return false;
-  }
-
-  return true;
-}
-
-function getCacheKeyForBudgetOrTransactionsSections(req, queryHash) {
-  if (
-    req.body.variables.kind &&
-    !isEqual(req.body.variables.kind.sort(), [
-      TransactionKind.ADDED_FUNDS,
-      TransactionKind.BALANCE_TRANSFER,
-      TransactionKind.CONTRIBUTION,
-      TransactionKind.EXPENSE,
-      TransactionKind.PLATFORM_TIP,
-    ])
-  ) {
+export function getGraphqlCacheProperties(req) {
+  if (req.remoteUser) {
     return;
   }
-  return `${req.body.operationName}_${queryHash}_${req.body.variables.slug}`;
-}
 
-export function getGraphqlCacheKey(req) {
-  if (req.remoteUser) {
+  // If it's not a GraphQL query with such properties, no cache
+  if (!req.body || !req.body.operationName || !req.body.query || !req.body.variables) {
```

Comment:
Can we also exclude mutations? Or maybe they never reach this check?

Reply:
I think we assume that Mutations are authenticated? And remoteUser is set?

But I guess we have some exceptions to this (guest flow, draft expenses), so would be worth checking.

Reply:
Added a check.

Reply:
> But I guess we have some exceptions to this (guest flow, draft expenses), so would be worth checking.

Also `createCollective`